### DISCUSSION TITLE
Fast Data | Control Plane start/stop features with Fast Data Components

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#571] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

Hi guys,
I'd like to know which features of the Control Plane works with which Fast Data flows/components.

* Event Driven flow &rarr; **RTU + SVTG + SVC** &rarr; the start/stop features works?
* Event Driven flow &rarr; **PS + SVTG + SVC** &rarr; the start/stop features works?
* Partial Event Driven flow &rarr; **RTU + SVTG (projections on MongoDB) + SVC** &rarr; the start/stop features works?
* Partial Event Driven flow &rarr; **PS + SVTG (projections on MongoDB) + SVC** &rarr; the start/stop features works?
* Standard Flow &rarr; **RTU (generates projections on MongoDB) + SVC** &rarr; the start/stop features works? The control plane works in read mode at least?
If I miss some use cases fell free to add it them.

I'm sorry I didn't find this information inside the documentation.

Thanks.

**Comment by ianFar96** [comment, marked as answer]
Hi there, the pause/resume functionality is independent on the architecture you use and it is implemented on the 4 main Fast Data services: RTU, PS, SVTG and SVC. The only binding you have is the version of each service which you can find here: https://docs.mia-platform.eu/docs/fast_data/runtime_management/compatibility_matrix#fast-data-services.

On a sidenote, it's worth mentioning that the [sv patch](https://docs.mia-platform.eu/docs/fast_data/concepts/architecture#standard-architecture-with-an-sv-patch) architecture is not yet supported

**Reply by frengooo** [reply]
Hi @ianFar96,
thanks for the reply 😄
